Wrappers/Python/setup.py

```diff
@@ -19,62 +19,62 @@
 
 
```

Comment:
Only change in this file is the entrypoints, and adding the ui module. The rest is an accidental formatting change